Лаба 2 ООП/Класс Library.py

```diff
@@ -0,0 +1,65 @@
+BOOKS_DATABASE = [
+    {
+        "id": 1,
+        "name": "test_name_1",
+        "pages": 200,
+    },
+    {
+        "id": 2,
+        "name": "test_name_2",
+        "pages": 400,
+    }
+]
+
+
+# TODO написать класс Book
+from typing import Union
+class Book:
+    def __init__(self, id: Union[int, float], name: Union[str], pages: Union[int, float]):
+        self.id = id
+        self.name = name
+        self.pages = pages
+
+    def __str__(self) -> str:
+        return f'Книга "{self.name}"'
+
+    def __repr__(self) -> str:
+        return f"Book(id_={self.id!r}, name={self.name!r}, pages={self.pages!r})"
+
+
+
+# TODO написать класс Library
+class Library:
+    def __init__(self, books: list = None):
+        if books is None:
+            self.books = []
+        else:
+            self.books = books
+
+    def get_next_book_id(self) -> int:
+        if not self.books:
+            return 1
+        else:
+            return self.books[-1]['id'] + 1
+
+    def get_index_by_book_id(self, id: int) -> int:
+        for index, book in enumerate(self.books):
+            if book['id'] == id:

```

Comment:
Аналогичная ситуация в этой строке. 
```suggestion
            if book.id == id:

```